docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Filesystem & Buckets
-description: dlt-verified source for Filesystem & Buckets
-keywords: [readers source and filesystem, files, filesystem, readers source, cloud storage]
+title: Cloud storage and file system
```

Comment:
Suggest we keep it the same as in the sidebar or change in both places :) 
```suggestion
title: Filesystem & cloud storage
```
<img width="844" alt="image" src="https://github.com/user-attachments/assets/12a4c658-dc54-4e22-95eb-a81a7010a4a1">

Reply:
You're right, I've updated the sidebar so now it's also consistent with the destinations